### PR TITLE
enum validation for pageId in mainpage component using zod, also upde…

### PR DESCRIPTION
**Pull request by ImmanuelCurrah**
…d ts.config to use stricmode true for this

### CODE REVIEW

File: TypeScript/src/App.tsx [hunk 1]
```diff
@@ -12,6 +12,8 @@ import Layout from "./ux-elements/Layout/Layout";
 
 export const pageContext = createContext<pageContextInterface>({
   pageIdentity: PageIdentityEnum.DEFAULT_PAGE,
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
```
Comment (ImmanuelCurrah) [inline TypeScript/src/App.tsx:15]: gotta figure this one out

File: TypeScript/src/ux-elements/footer-nav/nav.tsx [hunk 2]
```diff
@@ -41,7 +41,10 @@ export default function Nav() {
       </button>
       <button
         onClick={() => {
-          initializePage({ pageRoute: 4 }, page?.setPageNumber);
+          initializePage(
+            { pageRoute: PageIdentityEnum.FEED_PAGE },
+            page?.setPageIdentity
```
Comment (mike701) [inline TypeScript/src/ux-elements/footer-nav/nav.tsx:46]: do we need the ? here we took out from 16.

**Review by mike701 — APPROVED** [review]
LGTM! left one comment but merge at your leisure.

File: TypeScript/src/ux-elements/footer-nav/nav.tsx (hunk 2 above)
Comment (ImmanuelCurrah) [reply TypeScript/src/ux-elements/footer-nav/nav.tsx:46]: Ah yeah we probably dont need any of them, either one of us can take them out in the next pr

**Pull request merged**